packages/boba/gateway/src/services/networkService.js

```diff
@@ -2275,6 +2280,7 @@ class NetworkService {
   async getDaoVotes() {
 
     if( this.masterSystemConfig === 'mainnet' ) return
+    if( this.masterSystemConfig === 'rinkeby' ) return
```

Comment:
I guess we should return the empty object instead of nothing (undefined).

As in reducer on success response we are expecting the object to spread into the state.

https://github.com/omgnetwork/optimism-v2/blob/develop/packages/boba/gateway/src/reducers/daoReducer.js#L26


I will raise the issue for this.

Also should take care of this for other function as well.